tests/unit/test_virtualname.py

```diff
@@ -81,6 +81,8 @@ def _check_modules(self, path):
                     )
         return ret
 
+    @skipIf(not os.path.isdir(os.path.join(RUNTIME_VARS.CODE_DIR, 'salt/')),
```

Comment:
The slash should be removed from `salt/`

Reply:
Done.